database/index.js

```diff
@@ -4,8 +4,9 @@ const Promise = require('bluebird');
 
 const connection = mysql.createConnection({
   user: 'root',
+  password:'dobby',
```

Comment:
If the pw isn't specific to any personal pws then I guess its okay to leave it in here, but would be good practice to .gitignore any sensitive info.